### PR TITLE
Create initial standards & practices document

### PR DESCRIPTION
**Pull request by alexstojda**
Part of #5

### CODE REVIEW

**Review by danseremet — APPROVED** [review]
Looks very good to me.
One thing I would add to the documentation is for committing from HackMD. How would one leave their Github author name on the commits instead of HackMD?

I approve on the precondition of this minor change :+1:

**Comment by alexstojda** [comment]
> Looks very good to me.
> One thing I would add to the documentation is for committing from HackMD. How would one leave their Github author name on the commits instead of HackMD?
> 
> I approve on the precondition of this minor change 👍

Hm that's a limitation of HackMD sadly, we cannot change the commit user. Should we look at a different product? Or just revert to editing MD in an IDE & use a pair-programming tool? @danseremet

**Comment by danseremet** [comment]
We can do with a simple workaround. How about simply adding the github username to the commit message body?

**Comment by alexstojda** [comment]
> We can do with a simple workaround. How about simply adding the github username to the commit message body?
> 

Works for me! I'll update the document

File: standards-and-processes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+1. Documents not in HackMD (Skip if the document already exists):
+    1. New Document
+        1. Developer creates a new blank document
+    1. Existing Document in GitHub
+        1. Create a blank document & then select pull from GitHub.
+        2. Use the UI to select the file from GitHub
+1.  In the document, input required changes & commit changes when necessary by creating `versions` via the `Versions & GitHub Sync` menu. Ensure that your version/commit name follows commit naming conventions & **include the names of the persons who collaborated on that version.**
+2.  When ready to submit changes back to the repo, select `Push to Github` or `Push` and enter a new branch according to the naming conventions below, and push the changes. 
+3. Create a pull request & have it reviewed. If multiple people worked on the changeset, those persons cannot review this PR. 
+9. The pull reuquest must receive **at least 2 approving reviews**
```
Comment (danseremet) [inline standards-and-processes.md:63]: reuquest => request ?

Comment (alexstojda) [inline standards-and-processes.md:63]:
```suggestion
9. The pull request must receive **at least 2 approving reviews**
```

**Review by Alex-Hayeur — APPROVED** [review]
The standards are simple to follow but also look very helpful for keeping our repo understandable and prevent overwriting other's work. After the requested changes were made it looks good to adopt.

**Pull request merged**